tidb-cloud/manage-serverless-spend-limit.md

```diff
@@ -20,7 +20,7 @@ For the first five TiDB Serverless clusters in your organization, whether they a
 - Row-based storage: 5 GiB
 - [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit): 50 million RUs per month
 
-Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will not be affected. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.
+Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will not be affected, but will be throttled automatically. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.
```

Comment:
Once a cluster reaches its usage quota, it immediately denies any new connection attempts until you [increase the quota](#update-spending-limit) or the usage is reset upon the start of a new month. Existing connections established before reaching the quota will remain active but will experience throttling. For example, when the storage of a cluster exceeds 5 GiB for a free cluster, the cluster automatically restricts any new connection attempts.